lib/ansible/modules/git.py

```diff
@@ -241,57 +241,57 @@
 EXAMPLES = '''
 - name: Git checkout
   ansible.builtin.git:
-    repo: 'https://foosball.example.org/path/to/repo.git'
-    dest: /srv/checkout
+    repo: 'https://github.com/ansible/ansible.git'
+    dest: /tmp/checkout
     version: release-0.22
```

Comment:
```suggestion
    repo: https://github.com/ansible/ansible.git
    dest: /tmp/checkout
    version: stable-1.9
```

I think it's fine to retain some diversity of style to remind people that TMTOWTDI in YAML, but I don't like that every example of the HTTPS repository has the unnecessary quotes while every example of the SSH format is unquoted. This might cause readers to infer that these quotes are necessary in some way for this specific data, which is not true.